addons/hr_holidays/migrations/12.0.1.5/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,242 @@
+NEW ir.ui.view: hr_holidays.hr_leave_report_kanban
+NEW ir.ui.view: hr_holidays.hr_leave_report_tree
+NEW ir.ui.view: hr_holidays.hr_leave_view_calendar
+NEW ir.ui.view: hr_holidays.hr_leave_view_form
+NEW ir.ui.view: hr_holidays.hr_leave_view_form_manager
+NEW ir.ui.view: hr_holidays.hr_leave_view_kanban
+NEW ir.ui.view: hr_holidays.hr_leave_view_tree
+NEW ir.ui.view: hr_holidays.hr_leave_view_tree_my
+NEW ir.ui.view: hr_holidays.view_hr_holidays_filter_report
+NEW ir.ui.view: hr_holidays.view_hr_leave_allocation_filter
+DEL ir.ui.view: hr_holidays.edit_holiday_new
+DEL ir.ui.view: hr_holidays.hr_holidays_leaves_assign_tree_view
+DEL ir.ui.view: hr_holidays.view_holiday
+DEL ir.ui.view: hr_holidays.view_holiday_allocation_tree
+DEL ir.ui.view: hr_holidays.view_holiday_allocation_tree_customize
+DEL ir.ui.view: hr_holidays.view_holiday_employee
+DEL ir.ui.view: hr_holidays.view_holiday_new_calendar
+DEL ir.ui.view: hr_holidays.view_holiday_simple
+DEL ir.ui.view: hr_holidays.view_holiday_status_tree
+DEL ir.ui.view: hr_holidays.view_hr_holidays_kanban
+DEL ir.ui.view: hr_holidays.view_hr_holidays_remaining_leaves_user_graph
+DEL ir.ui.view: hr_holidays.view_hr_holidays_remaining_leaves_user_tree
+DEL ir.ui.view: hr_holidays.view_hr_holidays_status_search
+NEW mail.activity.type: hr_holidays.mail_act_leave_allocation_approval (noupdate)
+NEW mail.activity.type: hr_holidays.mail_act_leave_allocation_second_approval (noupdate)
+NEW mail.activity.type: hr_holidays.mail_act_leave_approval (noupdate)
+NEW mail.activity.type: hr_holidays.mail_act_leave_second_approval (noupdate)
+# NOTHING TO DO
+
+NEW mail.message.subtype: hr_holidays.mt_department_leave_allocation_approved (noupdate)
+NEW mail.message.subtype: hr_holidays.mt_department_leave_allocation_refused (noupdate)
+NEW mail.message.subtype: hr_holidays.mt_department_leave_approved (noupdate)
+NEW mail.message.subtype: hr_holidays.mt_department_leave_refused (noupdate)
+NEW mail.message.subtype: hr_holidays.mt_leave_allocation_approved (noupdate)
+NEW mail.message.subtype: hr_holidays.mt_leave_allocation_refused (noupdate)
+NEW mail.message.subtype: hr_holidays.mt_leave_approved (noupdate)
+NEW mail.message.subtype: hr_holidays.mt_leave_refused (noupdate)
+DEL mail.message.subtype: hr_holidays.mt_department_holidays_approved (noupdate)
+DEL mail.message.subtype: hr_holidays.mt_department_holidays_confirmed (noupdate)
+DEL mail.message.subtype: hr_holidays.mt_department_holidays_first_validated (noupdate)
+DEL mail.message.subtype: hr_holidays.mt_department_holidays_refused (noupdate)
+DEL mail.message.subtype: hr_holidays.mt_holidays_approved (noupdate)
+DEL mail.message.subtype: hr_holidays.mt_holidays_confirmed (noupdate)
+DEL mail.message.subtype: hr_holidays.mt_holidays_first_validated (noupdate)
+DEL mail.message.subtype: hr_holidays.mt_holidays_refused (noupdate)
+# DONE: it's preferable to not do the rename and delete the old ones instead
```

Comment:
Why not? You are then mangling all the followers preferences, and the messages with that subtypes becoming dangling.

Reply:
The problem is that it's a split, not a rename. `mt_holidays_approved` becomes both `mt_leave_approved` and `mt_leave_allocation_approved`. So should be rnamed to the first or to the second? Doesn't make sense.